test/serializers/developments_serializer_test.rb

```diff
@@ -1,11 +1,26 @@
 require 'test_helper'
 
 class DevelopmentsSerializerTest < ActiveSupport::TestCase
-  def serializer
-    @_base ||= DevelopmentsSerializer.new(
-      [developments(:one), developments(:two)]
+  def development_one
+    @_d1 ||= Development.create!(
+      id: 101010, name: 'Gadfly Hotel', address: '505 Washington Street', city: 'Boston', state: 'MA', zip_code: '02111', status: 'in_construction', commsf: 12, estemp: 75, private: true, created_at: Time.new('1969-12-31 19:00:00 -0500'), updated_at: Time.new('1969-12-31 19:00:00 -0500'), year_compl: 2016, creator: users(:normal)
+    )
+    @_d1.team_memberships = [DevelopmentTeamMembership.create(
+      development: @_d1, role: 'landlord', organization: organizations(:mapc)
+    )]
+    @_d1.save
+    @_d1
+  end
+
+  def development_two
+    @_d2 ||= Development.create!(
+      id: 101011, name: 'Hello', address: "It's me / I was wondering if after all these years", city: "you'd like to me", state: 'ET', zip_code: '02118', status: 'completed', commsf: 200, estemp: 1000, private: false, created_at: Time.new('1969-12-31 19:00:00 -0500'), updated_at: Time.new('1969-12-31 19:00:00 -0500'), year_compl: 2012, creator: users(:tim)
```

Comment:
Line is too long. [358/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.